templates/new_device_login.txt

```diff
@@ -1,6 +1,8 @@
 {{t "New sign-in to Firefox" }}
 
 {{ device }}
+{{t "IP address: %(ip)s" }}
+{{ location }}
```

Comment:
Even with an `if` around this line, the extra line appears if there is no location information. Not sure what the solution is.